src/Tiles/Providers/WmsCustomProvider.h

```diff
@@ -53,6 +55,8 @@ class WmsCustomProvider : public WmsProviderBase
 	void set_Format(CString value) { _format = value; }
 	tkWmsVersion get_Version() { return _version; }
 	void set_Version(tkWmsVersion value) { _version = value; }
+	tkWmsBoundingBoxOrder get_BoundingBoxOrder() { return _bbo; }
```

Comment:
I'm guessing we're just missing your updated IDL file, which would include the interface to these new methods.

Reply:
Oh, yes. I made a phased commit, then I forgot the IDL. Sorry! I've now committed it